pages/docs/reference/faq.md

```diff
@@ -9,7 +9,7 @@ title: FAQ
 
 ### What is Kotlin?
 
-Kotlin is an OSS statically typed programming language that targets the JVM, Android, JavaScript and Native. 
+Kotlin is an open-source statically-typed programming language that targets the JVM, Android, JavaScript and Native. 
```

Comment:
```suggestion
Kotlin is an open-source statically typed programming language that targets the JVM, Android, JavaScript, and Native. 
```

There is no hyphen in `statically typed` (see Microsoft or Google docs)

Reply:
Didn't know that. I used the text suggested on the kotlin forum, but I should have checked that before changing it. Not sure about the comma after JavaScript though. I'm not a native English speaker but I  don't remember every seeing a comma before an and.

Reply:
This comma is not mandatory but is widely used. It is called [_serial comma_](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) or Oxford comma.